src/comments.ts

```diff
@@ -128,7 +127,7 @@ export const RemindPeopleTheyCanUnblockPR = (user: string, approvalUsers: string
     status: txt`
         |:hourglass_flowing_sand: Hi @${user},
         |
-        |It's been a few days since this PR was approved by ${approvalUsers.join(", ")} and we're waiting for
+        |It's been a few days since this PR was approved and we're waiting for
```

Comment:
Actually I think this was already non-pinging, so maybe this change isn't necessary

Reply:
Correct, these are just usernames.